prow/config.yaml

```diff
@@ -6828,9 +6815,70 @@ periodics:
         secretName: ssh-key-secret
     - name: var-lib-docker
       emptyDir: {}
+- name: ci-ingress-gce-downgrade-e2e
```

Comment:
add a comment/TODO if this job does nothing yet, and assume you don't want this to run every hour now :-)

Reply:
Done. Changed to 24 hr